### DISCUSSION TITLE
Is there currently a way for s1 f4 chip users with warped beds to get a usable mesh from automesh

### DISCUSSION THREAD

**Discussion by MaynardMann** [mriscoc/Ender3V2S1#375] [Q&A]
Is there currently a way for s1 f4 chip users with warped beds to get a usable mesh from automesh without painstaking manual calibration? I am aware of the issue thread #195 here https://github.com/mriscoc/Ender3V2S1/issues/195 which many people go into length about their issues with the auto mesh with some suggested solutions but no concrete answers so I thought I'd make this thread. I tried everything proposed in issue #195 thread and nothing seems to make the automesh my machine generates usable in any degree without manually dialing in every prob point via edit mesh.

the s1 is my first home FDM printer Iv been in the industry of 3d printing a long time but my experience is with 10,000$+ machines owned by the company that ship with anatomically flat beds so I've never needed to deal with this issue or use a bed mesh so I could just be ignorant here is this the norm that all auto-generated meshes need to be drastically adjusted manually no matter machine or software or is this simply an issue with this software/hardware combination and if the latter is there currently a fix or are we waiting on an update to the firmware to hopefully fix it?

**Comment by mriscoc** [comment]
Some F4s have problems with the CRTouch, currently the only solution is using the CRTouch to create a mesh as reference and edit it manually.

**Reply by MaynardMann** [reply]
That's what I feared,  thank you for all your hard for the e3 community and activity on here and the other social media platforms answering questions my S1 f4's bed has a whopping 1.22mm deviation from the middle of the bed to the front left corner so if you are ever needing anyone to test compiled firmware to test potential fixes I'll gladly flash anything to this hunk of junk to see.

**Comment by hayden-t** [comment]
please see my 2 videos on the topic and how the problem can be fixed, nothing wrong with the firmware or chips. http://youtube.com/user/hthring
